test/provider_devdax_memory.cpp

```diff
@@ -14,6 +14,10 @@
 #include "cpp_helpers.hpp"
 #include "test_helpers.h"
 
+#define UMF_TEST_PROVIDER_FREE_NOT_SUPPORTED 1
+#include "ipcFixtures.hpp"
+#undef UMF_TEST_PROVIDER_FREE_NOT_SUPPORTED
```

Comment:
Instead of a macro, I suggest extending the `ipcTestParams` to contain a flag that tells if the provider supports free operation.

Reply:
Done